docs/how-to-install.md

```diff
@@ -6,145 +6,151 @@ license: "This software is licensed under the Apache License version 2."
+Gravitino uses [Log4j2](https://logging.apache.org/log4j/2.x/) as the logging system. The `conf/log4j2.properties` file provides the Gravitino server log configurations. To configure the Gravitino server log, you can update this file.
 
-For example, if you want to configure the Hive catalog, you can modify the file `catalogs/hive/conf/hive.conf`. The detailed configurations are listed in the specific catalog documentation.
+### Configure the Gravitino server environment variables
+
+Gravitino exposes several environment variables through the `conf/gravitino-env.sh` file. To configure these environment variables, you can update this file.
+
+### Configure the Gravitino catalogs
+
+Gravitino supports multiple catalogs. You can configure the catalog-level configurations by updating the related configuration file in the `catalogs/<catalog-provider>/conf` directory. For example, the `catalogs/hive/conf/hive.conf` file provides configurations for the Hive catalog. The configurations you set in the catalog configuration file will apply to all the catalogs of the same type that you created. For detailed configurations about each catalog, see related catalog documentation.
 
 :::note
-Gravitino takes the catalog configurations in the following order:
+Gravitino takes the catalog configurations in the following order of precedence:
 
-1. Catalog `properties` specified in catalog creation API or REST API.
+1. Catalog `properties` specified in the catalog creation API or REST API.
 2. Catalog configurations specified in the catalog configuration file.
 
-The catalog `properties` can override the catalog configurations specified in the configurationfile.
+The catalog `properties` overrides the catalog configurations specified in the configuration file.
 :::
 
-Gravitino supports passing in catalog-specific configurations if you add `gravitino.bypass.`. For example, if you want to pass in the HMS-specific configuration `hive.metastore.client.capability.check` to the underlying Hive client in the Hive catalog, add the `gravitino.bypass.` prefix.
+Gravitino supports passing in catalog-specific configurations by adding the `gravitino.bypass.` prefix to the specific catalog configurations. For example, you can pass in the HMS-specific configuration `hive.metastore.client.capability.check` to the underlying Hive client in the Hive catalog if you add the `gravitino.bypass.` prefix to the configuration.
 
-Also, Gravitino supports loading catalog-specific configurations from external files. For example,you can put your own `hive-site.xml` file in the `catalogs/hive/conf` directory, and Gravitino loads it automatically.
+Gravitino also supports loading catalog-specific configurations from external files. You just need to put your `hive-site.xml` file in the `catalogs/hive/conf` directory, and Gravitino will load it automatically.
```

Comment:
Where are these catalog-specific configurations located?
Is there shared location and/or naming convention?

Reply:
Good questions. This is my first time to contribute the doc. I am not familiar with the whole doc set. Maybe add it later when I go through the whole doc website. Is it OK?